zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```diff
@@ -135,6 +135,10 @@
           {
             "value": "other",
             "name": "Other"
+          },
+          {
+            "value": "part",
```

Comment:
Note, we're still pending a final decision on the name. Also, we have to ensure that treatments are not sent via direct indexing to OpenAIRE.